frontend/packages/lib-components/src/molecules/modals/InfoModal.tsx

```diff
@@ -171,7 +169,7 @@ function InfoModal({
                       <Button
                         onClick={reject}
                         dataQa="modal-cancelBtn"
-                        text={rejectLabel ?? i18n.common.cancel}
+                        text={rejectLabel ?? 'Peruuta'}
```

Comment:
Maybe these should now be mandatory props. Otherwise some Swedish-speaking user will eventually get furious.